backend/typescript/types.ts

```diff
@@ -37,6 +37,8 @@ export type CamperDTO = {
   chargeId: number;
 };
 
+export type CamperCSVInfoDTO = Omit<CamperDTO, "camps" | "id">;
```

Comment:
update camps -> camp since this is being changed on Camper model
```suggestion
export type CamperCSVInfoDTO = Omit<CamperDTO, "camp" | "id">;
```